src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = []
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
```

Comment:
If you want to pass linter tests, you should write all added properties with this and set the alive array in another way like this 
![image](https://user-images.githubusercontent.com/101983641/212717979-6dff24e1-959f-42f0-908e-fcb94b3326e8.png)